### PR TITLE
Check grouped buttons

### PR DESCRIPTION
**Pull request by liadove**
Need styling
some comments are left
image

### CODE REVIEW

**Comment by tesla809** [comment]
quick note:
1. can we have severe be the red-like color?
2. moderate yellow
3. blue-like color

Thoughts?

**Comment by ngiangre** [comment]
yes to more severe more red

**Comment by liadove** [comment]
something like gradient?
image

**Comment by ngiangre** [comment]
yes but diverging

File: client/src/components/survey-view/checkbox-button/CheckboxButton.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import SingleCheckboxButton from './SingleCheckboxButton';
+
+class CheckboxButton extends React.Component {
```
Comment (friedger) [inline client/src/components/survey-view/checkbox-button/CheckboxButton.js]: Could you make it a functional component use `useState` ?

Comment (liadove) [reply client/src/components/survey-view/checkbox-button/CheckboxButton.js]: like with hooks?

Comment (liadove) [reply client/src/components/survey-view/checkbox-button/CheckboxButton.js]: I can, but is this necessary?

Comment (SomeMoosery) [reply client/src/components/survey-view/checkbox-button/CheckboxButton.js]: Using functional components is typically best practices now with React - should be used as much as possible 

https://www.codeinwp.com/blog/react-best-practices/

Comment (friedger) [reply client/src/components/survey-view/checkbox-button/CheckboxButton.js]: useCallback would be useful here.

Comment (liadove) [reply client/src/components/survey-view/checkbox-button/CheckboxButton.js]: i used useReducer, that way I don't need useCallback

**Comment by SomeMoosery** [comment]
Looks good for now, if we need to make tweaks once V1 is up and running we can always come back.

Merging changes

**Comment by tesla809** [comment]
Amazing work @liadove !!!

**Comment by friedger** [comment]
Nice!!

**Pull request merged**